src/govuk/components/task-list/template.njk

```diff
@@ -0,0 +1,28 @@
+{% set idPrefix = params.idPrefix if params.idPrefix else "task-list" %}
+<ul class="govuk-task-list">
+  {% for item in params.items %}
+    {% if item.hint %}
+      {%- set hintId = idPrefix + "-" + loop.index + "-hint" -%}
+    {% endif %}
+    {%- set statusId = idPrefix + "-" + loop.index + "-status" -%}
+    <li class="govuk-task-list__item{% if item.href %} govuk-task-list__item--with-link{% endif %}">
+      <span class="govuk-task-list__task-name-and-hint">
+        {% if item.href %}
+          <a class="govuk-link govuk-task-list__link" href="{{ item.href }}" aria-describedby="{{ hintId + " " if item.hint }}{{ statusId }}">{{ item.title.html | safe if item.title.html else item.title.text }}</a>
+        {% else %}
+          <div class="govuk-task-list__task-no-link">{{ item.title.html | safe if item.title.html else item.title.text }}</div>
```

Comment:
`div` inside `span` is invalid HTML (for both `task-no-link` and `task-hint`)

Reply:
Updated now